packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -590,31 +594,18 @@ export class RunningSummarizer implements IDisposable {
 				this.beforeSummaryAction();
 			},
 			async () => {
+				if (this.mc.config.getBoolean("Fluid.Summarizer.TryDynamicRetries")) {
+					return this.trySummarizeWithDynamicRetries(reason, cancellationToken);
+				}
+
 				const attempts: (ISummarizeOptions & { delaySeconds?: number })[] = [
 					{ refreshLatestAck: false, fullTree: false },
 					{ refreshLatestAck: true, fullTree: false },
-					{ refreshLatestAck: true, fullTree: false, delaySeconds: 2 * 60 },
```

Comment:
We only do 2 attempts today so removing the code that deals with configuring and doing additional attempts.